app/views/papers/_tags_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="tags-form hidden">
+  <%= form_for(paper, html: {class: 'form-inline'}) do |f| %>
+    <div class="form-group col-md-12">
+      <%= f.text_field :tag_list, value: paper.tag_list.join(","), placeholder: "No tags added yet...", class: "form-control input-sm" %>
+      <%= f.submit 'Add tags', class: "btn btn-xs btn-primary" %>
+      <%= link_to "cancel", '#', class: "cancel-add-tag link-btn" %>
+    </div>
+  <% end %>
+  <i class='help-block'><small>Tags should be seperated by commas.</small></i>
+</div>
```

Comment:
Yeah, like that, but dumber. We'd basically just split it and create a new button whenever someone hits the space bar or the comma button